server/openapi.go

```diff
@@ -173,3 +176,60 @@ func (s *Server) findDatabase(ctx context.Context, host string, port string, dat
 
 	return nil, echo.NewHTTPError(http.StatusNotFound, fmt.Sprintf("Cannot find database %s in instance %s:%s", databaseName, host, port))
 }
+
+type schemaDiffRequestBody struct {
+	EngineType   parser.EngineType `json:"engineType"`
+	SourceSchema string            `json:"sourceSchema"`
+	TargetSchema string            `json:"targetSchema"`
+}
+
+// schemaDiff godoc
+// @Summary  Get the diff statement between source and target schema.
+// @Description  Parse and diff the schema statements.
+// @Accept  */*
+// @Tags  SQL review
+// @Produce  json
+// @Param  engineType  			body  string  true   "The database engine type."
+// @Param  sourceSchema     body  string  true   "The source schema statement."
+// @Param  targetSchema     body  string  false  "The target schema statement."
+// @Success  200  {array}   the target diff string of schemas
+// @Failure  400  {object}  echo.HTTPError
+// @Failure  500  {object}  echo.HTTPError
+// @Router  /sql/schema/diff  [post].
+func schemaDiff(c echo.Context) error {
+	request := &schemaDiffRequestBody{}
```

Comment:
Btw, we want to offer this API only to team/enterprise version.  Defer to @candy2255 to decide.